src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/BooleanToVisibilityConverter.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Windows;
+
+namespace NuGet.PackageManagement.UI.Converters
+{
+    internal class BooleanToVisibilityConverter
```

Comment:
Have you considered using built-in [BooleanToVisibilityConverter](https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.booleantovisibilityconverter?view=netframework-4.7.2) ?